index.bs

```diff
@@ -304,36 +303,72 @@ For the main <dfn>sanitize</dfn> operation, using a {{ParentNode}} |node|, a
          |attrName|:
          1. If "data-" is a [=code unit prefix=] of [=Attr/local name=] and
             if [=Attr/namespace=] is `null` and
-            if |config|["{{SanitizerConfig/dataAttributes}}"] exists and is false:
-            1. Remove |attr| from |child|.
-      1. else if |config|["{{SanitizerConfig/removeAttributes}}"] exists and
+            if |config|["{{SanitizerConfig/dataAttributes}}"] exists and is false,
+            then [=remove an attribute|remove=] |attr|.
+      1. Otherwise, if |config|["{{SanitizerConfig/removeAttributes}}"] exists and
          |config|["{{SanitizerConfig/removeAttributes}}"] [=SanitizerConfig/contains=]
-         |attrName|:
-         1. Remove |attr| from |child|.
+         |attrName|, then [=remove an attribute|remove=] |attr|.
       1. If |config|["{{SanitizerConfig/elements}}"][|elementName|] exists,
          and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/attributes}}"]
          exists, and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/attributes}}"]
-         does not [=SanitizerConfig/contain=] |attrName|:
-         1. Remove |attr| from |child|.
+         does not [=SanitizerConfig/contain=] |attrName|,
+         then [=remove an attribute|remove=] |attr|.
       1. If |config|["{{SanitizerConfig/elements}}"][|elementName|] exists,
          and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]
          exists, and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]
-         [=SanitizerConfig/contains=] |attrName|:
-         1. Remove |attr| from |child|.
-      1. If &laquo;[|elementName|, |attrName|]&raquo; matches an entry in the
-         [=navigating URL attributes list=], and if |attr|'s [=protocol=] is
-         "`javascript:`":
-         1. Then remove |attr| from |child|.
+         [=SanitizerConfig/contains=] |attrName|,
+         then [=remove an attribute|remove=] |attr|.
+      1. If the [=navigating URL attributes list=] [=SanitizerConfig/contains=]
+         &laquo;[|elementName|, |attrName|]&raquo; and |attr|
+         [=contains a javascript: URL=],
+         then [=remove an attribute|remove=] |attr|.
+      1. If |child|'s [=Element/namespace=] [=string/is=] the
+         [=MathML Namespace=] and |attr|'s [=Attr/local name=] [=string/is=]
+         "`href`" and  |attr| [=contains a javascript: URL=],
+         then [=remove an attribute|remove=] |attr|.
```

Comment:
(from our meeting) Maybe `xlink:href` is missing here...

Reply:
Added a check for the namespace (either null or XLink namespace).